bindings/jsmva/python/JsMVA/Utils.py

```diff
@@ -4,7 +4,4 @@
 #  Utilities module.
 
 import sys
-if sys.version_info >= (3, 0):
-    xrange = range
-else:
-    xrange = xrange
+xrange = range
```

Comment:
Can you not simply replace `xrange` with `range` in this source file? Then we don't need this alias